### PR TITLE
Bug 1502439: Add bug_1501329_affected to core ping

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

File: schemas/coverage/coverage/coverage.1.schema.json [hunk 1]
```diff
@@ -16,8 +16,11 @@
       "type": "string"
     },
     "telemetryEnabled": {
-      "type": "integer",
-      "enum": [0, 1]
+      "enum": [
```
Comment (fbertsch) [inline schemas/coverage/coverage/coverage.1.schema.json]: Did our formatting change? I can  remove this if needed.

Comment (mreid-moz) [reply schemas/coverage/coverage/coverage.1.schema.json]: Either the pretty-printing changed a bit or just didn't get checked in last time. There's [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1383111) on file to enforce consistency here.

Let's keep this fix, thanks!

File: templates/include/telemetry/coreCommon.2.schema.json [hunk 2]
```diff
@@ -47,4 +47,7 @@
   "items": {
     "type": "string"
   }
+},
+"bug_1501329_affected": {
```
Comment (mreid-moz) [inline templates/include/telemetry/coreCommon.2.schema.json]: Do we want to add this to all versions of the core ping? Or just the latest?

Comment (fbertsch) [reply templates/include/telemetry/coreCommon.2.schema.json]: We'll do all

**Pull request merged**